beta_rec/recommenders/lightgcn.py

```diff
@@ -0,0 +1,122 @@
+import os
+import time
+
+import numpy as np
+import torch
+from munch import munchify
+from ray import tune
+
+from ..data.deprecated_data_base import DataLoaderBase
+from ..models.lightgcn import LightGCNEngine
+from ..recommenders.recommender import Recommender
+from ..utils.monitor import Monitor
+
+
+def sparse_mx_to_torch_sparse_tensor(sparse_mx):
+    """Convert a scipy sparse matrix to a torch sparse tensor."""
+    sparse_mx = sparse_mx.tocoo().astype(np.float32)
+    indices = torch.from_numpy(
+        np.vstack((sparse_mx.row, sparse_mx.col)).astype(np.int64)
+    )
+    values = torch.from_numpy(sparse_mx.data)
+    shape = torch.Size(sparse_mx.shape)
+    return torch.sparse.FloatTensor(indices, values, shape)
+
+
+def tune_train(config):
+    """Train the model with a hypyer-parameter tuner (ray).
+
+    Args:
+        config (dict): All the parameters for the model.
+    """
+    data = config["data"]
+    train_engine = LGCN(munchify(config))
+    result = train_engine.train(data)
+    while train_engine.eval_engine.n_worker > 0:
+        time.sleep(20)
+    tune.track.log(
+        valid_metric=result["valid_metric"], model_save_dir=result["model_save_dir"],
+    )
+
+
+class LGCN(Recommender):
+    """The Matrix Factorization Mode."""
```

Comment:
- LGCN-->LightGCN, as it will be more clear?
- """The Matrix Factorization Mode.""" need to be updated.